Userland/Libraries/LibGUI/AbstractView.cpp

```diff
@@ -791,7 +791,10 @@ void AbstractView::drag_enter_event(DragEvent& event)
     //       We might be able to reduce event traffic by communicating the set of drag-accepting
     //       rects in this widget to the windowing system somehow.
     event.accept();
-    dbgln_if(DRAG_DEBUG, "accepting drag of {}", event.mime_types().first());
+
+    StringBuilder builder;
+    builder.join(',', event.mime_types());
+    dbgln_if(DRAG_DEBUG, "accepting drag of {}", builder.string_view());
```

Comment:
Building this string is wasted effort when `DRAG_DEBUG` is disabled. Since you're just concatenating the mime types, you can instead just pass the whole vector to the `dbgln` call:
```c++
dbgln_if(DRAG_DEBUG, "accepting drag of {}", event.mime_types());
```

Reply:
I felt icky doing this too, but was not aware printouts can accept data structures. Thanks for the review!

Reply:
Anything that has a `Formatter<T>` defined can be passed directly to printf-like functions like this.

Alternatively, you can wrap any extra code like this, and it'll be skipped if the debug flag isn't set:
```c++
if constexpr (DRAG_DEBUG) {
    do_this();
    do_that();
    dbgln("accepting drag");
}
```

Reply:
Makes perfect sense to me; thanks for your insight!